include/hal/library/cryptlib/cryptlib_cert.h

```diff
@@ -35,10 +35,10 @@ extern bool libspdm_asn1_get_tag(uint8_t **ptr, const uint8_t *end, size_t *leng
  * @param[in, out] subject_size  The size in bytes of the cert_subject buffer on input,
  *                               and the size of buffer returned cert_subject on output.
  *
- * @retval  true   The certificate subject retrieved successfully.
- * @retval  false  Invalid certificate, or the subject_size is too small for the result.
- *                 The subject_size will be updated with the required size.
- * @retval  false  This interface is not supported.
+ * @retval  true   If the subject_size is not euqal 0. The certificate subject retrieved successfully.
```

Comment:
All instances of `euqal` -> `equal`.

Reply:
Thanks. I have fixed all the typo for `equal`.